src/REPLMode/REPLMode.jl

```diff
@@ -284,7 +284,7 @@ end
 parse(input::String) =
     map(Base.Iterators.filter(!isempty, tokenize(input))) do words
         statement, _ = core_parse(words)
-        statement.spec === nothing && pkgerror("Could not determine command")
+        statement.spec === nothing && pkgerror("Could not determine command. Type ? for help with available commands")
```

Comment:
```suggestion
        statement.spec === nothing && pkgerror("$(first(split.(words, ' '))) is not a recognized command. Type ? for help with available commands")
```
(untested)

Reply:
Yeah. the unused `_` return from core_parse is actually what we want
```
(Pkg) pkg> version Foo
ERROR: `version` is not a valid command. Type ? for help with available commands

(Pkg) pkg> dsad
ERROR: `dsad` is not a valid command. Type ? for help with available commands

(Pkg) pkg> registry dsds
ERROR: `dsds` is not a valid command. Type ? for help with available commands
```

updated